# HG changeset patch
# User Daniel J Sebald <daniel.sebald@ieee.org>
# Date 1528086874 18000
#      Sun Jun 03 23:34:34 2018 -0500
# Node ID c7e67fcb86301d6b835bf7fa9d55cae47e1c25b6
# Parent  a17d388f35887000caba9d2eb5205cac893c873d
Supply missing refloat function symbol when QTBUG-44813 is ignored (bug #54051)

* variable-editor.cc (variable_dock_widget::refloat): Added.  Empty function
  for slot MOC when QTBUG-44813 workaround is not needed.

diff --git a/libgui/src/variable-editor.cc b/libgui/src/variable-editor.cc
--- a/libgui/src/variable-editor.cc
+++ b/libgui/src/variable-editor.cc
@@ -348,16 +348,20 @@ namespace octave
   }
 
 #else
 
   void
   variable_dock_widget::unfloat_float (void)
   {}
 
+  void
+  variable_dock_widget::refloat (void)
+  {}
+
 #endif
 
   // Variable editor stack
 
   variable_editor_stack::variable_editor_stack (QWidget *p)
     : QStackedWidget (p), m_edit_view (new variable_editor_view (this))
   {
     setFocusPolicy (Qt::StrongFocus);
